Regular-Question-Answers/lowest-common-ancestor-of-a-binary-tree.cpp

```diff
@@ -0,0 +1,60 @@
+// Author: Rojen Arda Şeşen
+// Question Link: https://leetcode.com/problems/lowest-common-ancestor-of-a-binary-tree/
+
+// We will first use preorder traversal to locate the given nodes.
+// While traversing, we will store the path to nodes using a vector.
+// Each time we visit a node, we will push it to the vector.
+// We will pop the node from the vector when returning from it.
+// We will save the vector when we encounter one of the nodes we are searching for.
```

Comment:
Maybe you can give a little more detail. you can mention saving the paths in two different global vectors